js/main.js

```diff
@@ -1,2 +1,125 @@
 $(function () {
+
+// Rock Paper Scissor routine input user choice, generate random computer response
+// find out who won, update score and then present result in table on screen
+// There seems to be a bias in the program toward the user -- cause not clear
+// First define globale variables that are used to display results or simplify flow:
+  let userWins=0
+  let machineWins=0
+  let machineInput=NaN
+  let userInput=NaN
+
+  // The evenhandler is where we are waiting for input
+  // Very much in contrast to "normal linear programming"
+  // Everything emenates from here
+
+  // If the user clicks "rock":
+  $('#rock').click(() => {
+  let userInput = $('#rock').text()
+  mainLoop(userInput)
+  })
+  // If the user clicks "paper":
+  $('#paper').click(() => {
+    let userInput = $('#paper').text()
+    mainLoop(userInput)
+  })
+  // If the user clicks "scissors":
+  $('#scissors').click(() => {
+    let userInput = $('#scissors').text()
+    mainLoop(userInput)
+  })
+
+  //The mainLoop is where the action is after we have received user input
+  function mainLoop(userInput){
+    // first get the input from the machine into "MachineInput"
+    getMachineInput()
+    // find out who wins using the function from previous home work
+    winner=findWinner(userInput,machineInput)
+    // Depending on who wins, update total scores
+    updateScores(winner)
+    // change score board
+    $('#humanScore').text(userWins)
+    $('#computerScore').text(machineWins)
+    // Extra info for the user for clarity and also debugging -- a little awkward but it works
+    if (winner===1) {gwin="User"} else {gwin="Bot"}  //transfer winner to text that we can use
+    $('#status').html("Computer played " + '<u>'+machineInput+'</u>' + '.' + '\n'
+    + "You played " + '<u>'+userInput+'</u>'+'.' + '</br>' + gwin + " Won" )
```

Comment:
@AtleLohrmann the UI does not account for tied games

check solution [here](https://github.com/jsr20190826/rock-paper-scissors-solution/blob/1583be48146e2521b25308e017c52fbfcf0d9d89/js/main.js#L82)